homeassistant/components/zha/core/channels/__init__.py

```diff
@@ -392,17 +404,3 @@ def cluster_command(self, tsn, command_id, args):
             self.zha_send_event(
                 self._cluster, self._cluster.server_commands.get(command_id)[0], args
             )
-
-
-# pylint: disable=wrong-import-position, import-outside-toplevel
-from . import closures  # noqa: F401
-from . import general  # noqa: F401
```

Comment:
These need to stay here #29555

Reply:
Hehe these should not even be in this PR 😉 

@basnijholt Could you remove the one component that started with "Z" from this PR 😉 👍

Reply:
Fixed!